[2024-07-22T09:03:59.105+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-22T09:03:59.113+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.get_src_tables manual__2024-07-22T09:03:57.109455+00:00 [queued]>
[2024-07-22T09:03:59.116+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.get_src_tables manual__2024-07-22T09:03:57.109455+00:00 [queued]>
[2024-07-22T09:03:59.117+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-22T09:03:59.121+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_src_tables> on 2024-07-22 09:03:57.109455+00:00
[2024-07-22T09:03:59.126+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=4984) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-22T09:03:59.127+0000] {standard_task_runner.py:64} INFO - Started process 4995 to run task
[2024-07-22T09:03:59.128+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'get_src_tables', 'manual__2024-07-22T09:03:57.109455+00:00', '--job-id', '522', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpmbvqw6lx']
[2024-07-22T09:03:59.128+0000] {standard_task_runner.py:91} INFO - Job 522: Subtask get_src_tables
[2024-07-22T09:03:59.145+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.get_src_tables manual__2024-07-22T09:03:57.109455+00:00 [running]> on host d89911194c67
[2024-07-22T09:03:59.173+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='get_src_tables' AIRFLOW_CTX_EXECUTION_DATE='2024-07-22T09:03:57.109455+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-22T09:03:57.109455+00:00'
[2024-07-22T09:03:59.174+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-22T09:03:59.176+0000] {base.py:84} INFO - Using connection ID 'postgres_server_2' for task execution.
[2024-07-22T09:03:59.181+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:269: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2024-07-22T09:03:59.199+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-22T09:03:59.199+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-22T09:03:59.202+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=get_src_tables, run_id=manual__2024-07-22T09:03:57.109455+00:00, execution_date=20240722T090357, start_date=20240722T090359, end_date=20240722T090359
[2024-07-22T09:03:59.226+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-22T09:03:59.237+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-22T09:03:59.238+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
